src/RequestBuilder/RequestBuilderFactory.php

```diff
@@ -46,6 +46,15 @@ public function events(): EventsRequestBuilder
         return $this->createConfiguredBuilder(EventsRequestBuilder::class);
     }
 
+    /**
+     * @param string[] $itemIds
+     * @return SortingRequestBuilder
+     */
+    public function sorting(string $userId, array $itemIds): SortingRequestBuilder
```

Comment:
What about passing Sorting command instance to this method instead? As shown here: https://github.com/lmc-eu/matej-client-php/issues/36

This introduces another way of creating Sorting command... And its also unsystematic - all methods of request Builders accept Command instances, there is no other "shortcut" for direct instantiation...